docs/content/latest/api/ysql/the-sql-language/statements/ddl_refresh_matview.md

```diff
@@ -0,0 +1,101 @@
+---
+title: REFRESH MATERIALIZED VIEW statement [YSQL]
+headerTitle: REFRESH MATERIALIZED VIEW
+linkTitle: REFRESH MATERIALIZED VIEW
+description: Use the REFRESH MATERIALIZED VIEW statement to refresh the contents of a materialized view.
+menu:
+  latest:
+    identifier: ddl_refresh_matview
+    parent: statements
+aliases:
+  - /latest/api/ysql/commands/ddl_refresh_matview/
+isTocNested: true
+showAsideToc: true
+---
+
+## Synopsis
+
+Use the `REFRESH MATERIALIZED VIEW` statement to replace the contents of a materialized view.
+
+## Syntax
+
+<ul class="nav nav-tabs nav-tabs-yb">
+  <li >
+    <a href="#grammar" class="nav-link active" id="grammar-tab" data-toggle="tab" role="tab" aria-controls="grammar" aria-selected="true">
+      <i class="fas fa-file-alt" aria-hidden="true"></i>
+      Grammar
+    </a>
+  </li>
+  <li>
+    <a href="#diagram" class="nav-link" id="diagram-tab" data-toggle="tab" role="tab" aria-controls="diagram" aria-selected="false">
+      <i class="fas fa-project-diagram" aria-hidden="true"></i>
+      Diagram
+    </a>
+  </li>
+</ul>
+
+<div class="tab-content">
+  <div id="grammar" class="tab-pane fade show active" role="tabpanel" aria-labelledby="grammar-tab">
+    {{% includeMarkdown "../../syntax_resources/the-sql-language/statements/refresh_matview.grammar.md" /%}}
+  </div>
+  <div id="diagram" class="tab-pane fade" role="tabpanel" aria-labelledby="diagram-tab">
+    {{% includeMarkdown "../../syntax_resources/the-sql-language/statements/refresh_matview.diagram.md" /%}}
+  </div>
+</div>
+
+## Semantics
+
+Replace the contents of a materialized view named *matview_name*. 
+
+The default mechanism for refreshing a matview is loading the updated contents in a new relation, and then swapping the matview's
+storage with that of the new relation's.
+### WITH DATA
+If `WITH DATA` (default) is specified, the view's query is executed to obtain the new data and the materialized view's contents are updated.
+
+### WITH NO DATA
+If `WITH NO DATA` is specified, the old contents of the materialized view are discarded and the materialized view is left in an unscannable state.
+
+### CONCURRENTLY
+This option differs in the mechanism used to perform the refresh. When `CONCURRENTLY` is specified, a diff between the old data and the new data is computed, and the required updates are performed accordingly. 
```

Comment:
Ideally we should also  specify or connect the dots to specify  the scenario when concurrently is performs better  and when it does not.
From PG docs  -->
This option may be faster in cases where a small number of rows are affected. Without this option a refresh which affects a lot of rows will tend to use fewer resources and complete more quickly